### ISSUE TITLE
Fix tests failure with Chrome 71

### ISSUE THREAD

**Issue by zhouyx** [ampproject/amphtml#19647]

**Comment by choumx** [comment]
Fixed the Travis failures in #19658. 

Saucelabs just upgraded to 71 too and different (integration) tests are failing: https://travis-ci.org/ampproject/amphtml/jobs/464777879

**Comment by choumx** [comment]
Looks like they're mostly amp-form integration tests. I'll fix the single amp-bind one.

/to @cvializ

**Comment by cvializ** [comment]
Looks like the amp-form integration test was due to a race. Fixed in #19824

**Comment by choumx** [comment]
Chrome 71 also caused a different set of tests to flake on SauceLabs (on Windows 10): 

```
DESCRIBE => amp-carousel
  DESCRIBE => type=carousel with multiple images
    IT => "before each" hook for "should not be able to go past the first or last item"
      ✗ Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

    IT => "before each" hook for "should only have the next button enabled when on first item"
      ✗ Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

DESCRIBE => error page
  IT => "before each" hook for "should show the body in error test"
    ✗ Error: Failed to find errors. HTML

  IT => "before each" hook for "should fail to load #iframe0"
    ✗ Error: Failed to find errors. HTML

DESCRIBE => amp-video
  DESCRIBE => Analytics Triggers
    IT => should trigger session analytics when a session ends
      ✗ Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

DESCRIBE => Viewer Visibility State
  DESCRIBE => Element Transitions
    IT => "before each" hook for "does not call callbacks when going to PAUSED"
      ✗ Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

DESCRIBE => user-error
  DESCRIBE => 3p user-error integration test
    DESCRIBE =>  
      IT => should ping correct host with 3p error message
        ✗ Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

DESCRIBE => amp-image-slider
  DESCRIBE => amp-image-slider
    IT => "before each" hook for "should apply custom styling on labels and hints"
      ✗ Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```
